packages/browser/src/dpop/__mocks__/DpopClientKeyManager.ts

```diff
@@ -34,3 +34,13 @@ export const DpopClientKeyManagerMock: jest.Mocked<IDpopClientKeyManager> = {
     async () => DpopClientKeyManagerMockGetClientKeyResponse
   ),
 };
+
+export const mockDpopClientKeyManager = (
+  key: JSONWebKey | undefined
+): IDpopClientKeyManager => {
+  return {
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    generateClientKeyIfNotAlready: async (): Promise<void> => {},
```

Comment:
To actually match the type signature:

```suggestion
    generateClientKeyIfNotAlready: async (): Promise<void> => undefined,
```

Somewhat shorter:

```suggestion
    generateClientKeyIfNotAlready: () => Promise.resolve(),
```